src/connected-components/TextArea/index.tsx

```diff
@@ -0,0 +1,7 @@
+import * as React from 'react';
+import styles from './TextArea.st.css';
+import { TextArea as TextAreaOrig } from '../../components/TextArea';
+
+export const TextArea = props => (
+  <TextAreaOrig {...props} {...styles('root', {}, props)} />
+);
```

Comment:
Why do we need both `connected-components/TextArea/index.tsx` and `connected-components/TextArea/index.example.tsx` files?
They seem to be the same.
And also, would rather have these files as part of the generated files in the `TextArea` folder, as previously commented